api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -92,6 +98,48 @@ public synchronized Order saveOrder(Order order, OrderContext orderContext) thro
 	}
 
 	/**
+	 * @see org.openmrs.api.OrderService#saveOrders(Set)
+	 */
+	public synchronized Set<Order> saveOrders(Set<Order> orders) {
```

Comment:
As part of the new discussion, we have removed the OrderGroup from the Encounter.

if there were orders which were independently ordered they were getting saved normally.

if there were orders which had orderGroups, we used to save the orderGroup first, and then individually save the orders present in the orderGroup.

This was a bit inconsistent. So we changed the approach to just save orders.
If it has orderGroup, it will be saved,
or else the order will be independently saved.